latest/src/app/documentation/demos/full-page-layouts/basepage-layout/basepage-layout.demo.html

```diff
@@ -2,18 +2,33 @@
     <clr-demo-menu></clr-demo-menu>
 
     <div class="content-header">
-        <h2><span *ngIf="!withCommandBar">Base Page Layout</span> 
-            <span *ngIf="withCommandBar">Command bar</span></h2>
+        <h2><span *ngIf="!withCommandBar && !withContentPanel">Base Page Layout</span>
+            <span *ngIf="withCommandBar && !withContentPanel">Command bar</span>
+            <span *ngIf="withCommandBar && withContentPanel">Content Panel</span></h2>
         <clr-button-group class="command-bar" [clrMenuPosition]="'bottom-right'" *ngIf="withCommandBar">
-            <clr-button>Command1</clr-button>
-            <clr-button>Command2</clr-button>
+            <clr-button *ngIf="!withContentPanel">Command1</clr-button>
+            <clr-button *ngIf="!withContentPanel">Command2</clr-button>
+            <clr-button *ngIf="withContentPanel" (click)="toggleLeftPanel()">Show/Hide Left</clr-button>
+            <clr-button *ngIf="withContentPanel" (click)="toggleRightPanel()">Show/Hide Right</clr-button>
             <clr-button [clrInMenu]="true">Command3</clr-button>
             <clr-button [clrInMenu]="true">Command4</clr-button>
-        </clr-button-group> 
+        </clr-button-group>
     </div>
-    <div class="content-container">
+    <div class="content-container" *ngIf="!withContentPanel">
         <div class="content-area">
             This is the page content
         </div>
     </div>
+    <clr-content-panel-container *ngIf="withContentPanel">
```

Comment:
This component should have class content-container attached by default.
And inside the component where the class "content-panel-outer-content" is attached, also the class "content-area" should be added.
This way the first child will have it's margin-top set to 0 by clarity default css.
Then you may also cleanup some css rules which overlap in those classes.
Sorry for that late finding, but just saw it in the demo.